.circleci/config.yml

```diff
@@ -662,13 +627,9 @@ jobs:
             fi
           when: on_fail
       - run:
-          name: 'Copy coredumps'
+          name: 'Print stack traces'
           command: |
-            mkdir -p /tmp/core_dumps
-            core_files=( $(find . -type f -regex .*core.*\d*.*postgres) )
-            if [ ${#core_files[@]} -gt 0 ]; then
-                cp "${core_files[@]}" /tmp/core_dumps
-            fi
+            ./ci/print_stack_trace.sh
```

Comment:
Would be good to install postgres-debug packages before printing the stack trace, so that stack traces are more useful.